Lessons12/HomeWork.py

```diff
@@ -0,0 +1,30 @@
+"""
+HomeWork 12.
+
+Given list of list of list etc of integers
+write recursive function that will accept
+as argument target list and return sum of
+all integers inside it
+Input:
+[[[[1, 4, 5], [[6, 9],[[[8, 1], 7], 3], 2], 7], 5, 2], 9, [1, 2]]
+Output: Target sum = 72
+"""
+
+
+y = [[[[1, 4, 5], [[6, 9], [[[8, 1], 7], 3], 2], 7], 5, 2], 9, [1, 2]]
+
+
+def sum_lists_values(list_with_values: list, count: int = None) -> int:
```

Comment:
In general solution correct , but count as parameter looks a little bit redundant here.
As advanced level try to examine this example:
```
def complex_list_sum(list_):
    curr_sum = 0
    for el in list_:
        curr_sum += complex_list_sum(el) if type(el) is list else el
    return curr_sum
```

Reply:
Yep/ looks bettter